trashtalk/templates/index.html

```diff
@@ -11,9 +11,6 @@
 <div class="container">
     <div class="row text-center">
         <h1>Welcome to Trash Talk</h1>
-        {% if not user.is_authenticated %}
```

Comment:
Why? I think we should definitely show context appropriate links (if a user is logged in, they should not see a signup or login link).